src/pages/CheckReservation/CheckReservation.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import ReservationTicket from './ReservationTicket';
+import S from './Styled.CheckReservation';
+
+const CheckReservation = () => {
+  const [reservationData, setReservationData] = useState([]);
+  const [currentTab, setCurrentTab] = useState('예정된 여행');
+
+  useEffect(() => {
+    fetch('/data/Reservation_Data-1.json')
+      .then(res => res.json())
+      .then(data => setReservationData(data.result));
+  }, []);
+
+  const handleCurrentTab = e => {
+    const currentTabName = e.target.innerHTML;
+    const currentTabId = e.target.id;
+    fetch(`/data/Reservation_Data-${currentTabId}.json`)
+      .then(res => res.json())
+      .then(data => setReservationData(data.result));
+    setCurrentTab(currentTabName);
+  };
```

Comment:
```suggestion
  const handleCurrentTab = (id, title) => {
    fetch(`/data/Reservation_Data-${id}.json`)
      .then(res => res.json())
      .then(data => setReservationData(data.result));
    setCurrentTab(title);
  };
```
- id와 title를 받아서 진행해도 되겠네요.